### PR TITLE
Add PHPStan, Travis and PSR-4 autoload

### PR DESCRIPTION
**Pull request by TomasVotruba**
**Merge only after #4 !**

Travis is now running! https://travis-ci.org/eventigo/eventigo-web

+ nice badge added to README :)

See 

- https://www.tomasvotruba.cz/blog/2017/01/28/why-I-switched-scrutinizer-for-phpstan-and-you-should-too/
- https://zlml.cz/psr-4-autoloader-aplikace
- https://github.com/pehapkari/pehapkari.cz/pull/207/files#diff-16226809d57b6a10005134f8f8ae9df3

### CODE REVIEW

**Review by filipsuk — APPROVED** [review]

**Review by jakubzaba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thanks guys, you're fast!

File: app/Modules/Newsletter/Model/NewsletterService.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ class NewsletterService
 	public $linkGenerator;
 
 	/** Path to css file used for css inline of newsletter texts html */
-	const CSS_FILE_PATH = __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'presenters' . DIRECTORY_SEPARATOR . 'templates' . DIRECTORY_SEPARATOR . 'Newsletter' . DIRECTORY_SEPARATOR . 'build.css';
```
Comment (TomasVotruba) [inline app/Modules/Newsletter/Model/NewsletterService.php:69]: This was accidentally removed. I'll fix that!

Comment (TomasVotruba) [reply app/Modules/Newsletter/Model/NewsletterService.php:69]: It's ok on master branch and my local master branch. Interesting :) OK
https://github.com/eventigo/eventigo-web/blob/master/app/modules/newsletter/model/NewsletterService.php#L69

**Comment by TomasVotruba** [comment]
This branch is merged, but in master missing. Any idea what happened?

**Comment by TomasVotruba** [comment]
Oh, I see. This was supposed to be merged before #4, my bad. I fill fix it :)

**Pull request merged**